examples/src/main/java/com/ibm/watson/speech_to_text/v1/MicrophoneWithWebSocketsExample.java

```diff
@@ -16,6 +16,7 @@
 import com.ibm.cloud.sdk.core.security.Authenticator;
 import com.ibm.cloud.sdk.core.security.IamAuthenticator;
 import com.ibm.watson.speech_to_text.v1.model.RecognizeOptions;
```

Comment:
I don't think you need this import anymore

Reply:
good catch!